### DISCUSSION TITLE
PR review show changes since your last review

### DISCUSSION THREAD

**Discussion by Eryx5502** [pwntester/octo.nvim#458] [Q&A]
I just started using octo.nvim for PR reviewing, coming from github web interface, and there is something I'm missing. When working from github web interface if you're doing a revision on a PR you already reviewed before, there's an option to "show changes since your last review" and also one for showing changes from specific selection of commits.

Is this possible with octo.nvim?

**Comment by andy941** [comment]
I second that, this is one of the reasons why I still need to use the webui. I know we have the commit-wise review but it is not the same thing. Most of the code I review is not well organized in sel-contained commits but it has a messy history of doing one thing and rewriting it. It should not be this way but when I'm on the job it's the sad reality of things. This feature would help a lot.

**Comment by williambdean** [comment]
Is this currently possible @GuillaumeLagrange ?

**Reply by GuillaumeLagrange** [reply]
This is not in octo, but something that would actually make a lot of sense.

First step would be to take a look at the graphql api and what it offers around this. I'll post updates here and create a dedicated issue when I get the time to do so.

Unfortunately lately I have not had much free time so if anyone wants to tackle this they're welcome to do so.
